frontend/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const createProxyMiddleware = require('http-proxy-middleware');
+
+module.exports = function (app) {
+    app.use(
+        '/admin',
+        createProxyMiddleware({
+            target: 'http://localhost:8000',
+            changeOrigin: true,
+        })
+    );
+};
```

Comment:
```suggestion
};

```

Not sure how strict you are on EOF newlines.